test/controllers/choices_controller_test.rb

```diff
@@ -12,8 +12,27 @@ class ChoicesControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should get edit" do
+<<<<<<< HEAD
```

Comment:
There are conflicts here.
The conflicts look like this

**<<<<<<< HEAD** 

Please fix.